docs/rules/no-array-constructor.md

```diff
@@ -38,6 +38,12 @@ Array(500)
 new Array(someOtherArray.length)
 ```
 
+```js
```

Comment:
Can we combine these two examples into one code block and do the same for the incorrect examples while we're at it?

Reply:
Done.